brave-intl/publishers
# update brave-ui, copy old rewards banner from old brave-ui

Update brave-ui to fix https://github.com/brave-intl/publishers/security/dependabot/148

Brave-ui has not had any versions published to npm in a few years. Instead we go direct to github. The latest version does not have non-component UI, e.g. Rewards banners, etc. That is contained within brave-core itself now. So I brought in that old code to this app as it was out of date and a different version and design to what's in the browser now.


#### How To Test
Check site banner preview still works, and all pages which use brave-ui components

#### Pull Request Checklist
